### ISSUE TITLE
Warlocks getting extra invocations (too many)

### ISSUE THREAD

**Issue by clifton-weatherford** [SolastaMods/SolastaUnfinishedBusiness#2406]
My three warlocks seem to be getting bonus invocations for no reason. Three screenshots attacks. 

1. Rogue4/Warlock9 - When I hit 9th Level in Warlock, I got two extra.
2. Bard9/Warlock2/Sorc1 - When I hit 10th level BARD, I got four extra.
3.  Sorc13/Warlock1 - When I took my first level of Warlock, I got SIXTY SIX extra. 
![Warlock_extra_invokations_SorLock](https://user-images.githubusercontent.com/124588122/218227413-ab420353-b240-462f-a819-68a9cf2f04ff.png)
![Warlock_extra_invokations_BardLock](https://user-images.githubusercontent.com/124588122/218227416-a76cee14-bb52-4ba0-970f-4dc67a2e002f.png)
![Warlock_extra_invokations_RogueLock](https://user-images.githubusercontent.com/124588122/218227417-616f777d-9666-4861-8a05-180a8abc196f.png)

**Comment by clifton-weatherford** [comment]
Addition comment: When my Sorlock cancelled out of level up and tried to go back to Sorcerer, it still tried to give him invocations, 78 this time. When I cancelled completely out of level up and then back to level up and picked Warlock again, I got 12 invocations. I cancelled out of level up completely, then went back in, chose sorcerer this time (no warlock levels at all), the invocations were no longer displayed.
